slider/js/script.js

```diff
@@ -0,0 +1,62 @@
+const slider = (function() {
```

Comment:
Call modules only in Upper camel case format https://ru.wikipedia.org/wiki/CamelCase